packages/jsii/lib/assembler.ts

```diff
@@ -143,12 +145,23 @@ export class Assembler implements Emitter {
 
     async function _loadReadme(this: Assembler) {
```

Comment:
why are those not simple private methods?

Reply:
The prototype of this class is already... quite heavily polluted. I did not want to add (more) clutter.